README.md

```diff
@@ -1,85 +1,133 @@
-<h1 align="center">
-    <a target="_blank" href="https://djangoindia.org/home"> 
-        <img src="https://djangoindia.org/whatIsDjango/Logo.svg">
-    </a>
-</h1>
+<div style="align: center;">
+  <img src="assets/webiste-front.png" alt="Django India Logo" />
+</div>
```

Comment:
ummm I think it's not better to put website front page at top :/

Reply:
changed